examples/remix-auth-auth0/app/routes/private.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ActionFunction, LoaderFunction } from "remix";
+import { Form, json, useLoaderData } from "remix";
+import type { Auth0Profile } from "remix-auth-auth0";
+import { auth } from "~/utils/auth.server";
+
+type LoaderData = { profile: Auth0Profile };
+
+export const action: ActionFunction = async ({ request }) => {
```

Comment:
I think this is not needed, for logout there’s another URL

Reply:
It was implemented incorrectly. I fixed it.